### DISCUSSION TITLE
Show repository size

### DISCUSSION THREAD

**Discussion by ravener** [community/community#34926] [Mobile]
Suggestion to show repository's total size. It's really useful for me to know how big a repository is before cloning it. FastHub app shows this and so it'd be great to have it in the official app.

**Comment by HenSquared** [comment]
Hi @ravener!

I appreciate you taking the time to make this feature request. 

I've added this feedback to our internal tracking issue. We will continue to watch for similar feature requests and prioritize work accordingly. 

And thanks for using the app!
